rpcs3/CMakeLists.txt

```diff
@@ -313,11 +313,7 @@ if(NOT LLVM_FOUND)
 else()
 	add_definitions(${LLVM_DEFINITIONS})
 	add_definitions(-DLLVM_AVAILABLE)
-	if(CMAKE_BUILD_TYPE STREQUAL "Release")
-		llvm_map_components_to_libnames(LLVM_LIBS core support executionengine object runtimedyld x86desc x86info scalaropts mcjit vectorize ipo x86codegen x86disassembler)
```

Comment:
You most likely didn't link the libraries in the right order previously or were missing some targets. This is using the main targets defined by the LLVM CMake build system and should be correct.
It will pull the right dependencies in the right order on the linker command line.